Sources/SwiftDocC/Model/ParametersAndReturnValidator.swift

```diff
@@ -664,6 +664,18 @@ struct ParametersAndReturnValidator {
 
 // MARK: Helper extensions
 
+private extension Return {
+    /// Applies a basic heuristic to give an indication if the return value documentation possibly documents what happens when an error occurs
+    func possiblyDocumentsFailureBehavior() -> Bool {
+        contents.contains(where: { markup in
+            let formatted = markup.format().lowercased()
+            // Check if the authored markup contains one of a handful of words as an indication that it possibly documents what happens when an error occurs.
+            return ["error", "`nil`", "fails", "failure"].contains(where: { word in
```

Comment:
Should we also allow an unformatted `nil` value?
```suggestion
            return ["error", "`nil`", "nil", "fails", "failure"].contains(where: { word in
```

Reply:
I also wonder whether we should check that these are the full word and not part of a word. For example, what if the return documentation contained one of the following words:
> failsafe 
> error-prone

Reply:
I added tests for this in https://github.com/swiftlang/swift-docc/pull/1096/commits/30bf06408039d4399ff5029e352efc8dd12e915a